### DISCUSSION TITLE
SSL Error in Janssen

### DISCUSSION THREAD

**Discussion by 0786ShivaniBansal** [JanssenProject/jans#5633] [Q&A] [answered]
Hii I'm trying to test agama in janssen server while running python3.10 main.py to test agama I'm getting SSL error
SSLError
requests.exceptions.SSLError: HTTPSConnectionPool(host='server-l.us.corp.sennovate.com', port=443): Max retries exceeded with url: /.well-known/openid-configuration (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'server-l.us.corp.sennovate.com'. (_ssl.c:1007)")))

Hoping to get resolve this soon please reply as soon as possible

**Comment by ossdhaval** [comment]
Hi @0786ShivaniBansal Prima facie, looks like issues with the certificate installed. `Hostname mismatch`

Adding @jgomer2001 for more comments.

**Comment by jgomer2001** [comment]
it is unclear what you are trying to do, and when and where this error happens. main.py script isn't familiar to me either

**Comment by nynymike** [comment, marked as answer]
The error is pretty self explanatory... your client can't verify the self-signed cert of your Jans Auth Server. Did you import the self signed cert into your client properly? Perhaps you should use Jans Tarp as your test RP.  See the [video](https://www.loom.com/share/6bfe8c5556a94abea05467e3deead8a2) I recorded.
